authui/src/authflowv2/base.css

```diff
@@ -47,6 +47,12 @@
     --border-radius-rounded-square: 14px;
   }
 
+  /* Ring Width */
+  :root {
+    --ring-width-0: 0px;
+    --ring-width-1\.5: 1.5px;
```

Comment:
1. I  think we don't want to name a css variable using the actual value. Maybe we should call it something like `--ring-width-common`.
2. Maybe ring width is a special case that we can directly write `1.5px` as the default value in the component, and don't need a variable in base styles, because ring is unlikely to be changed together with other components. @louischan-oursky Also see how louis think

Reply:
Let's write `ring-width-[1.5px]` directly on the component HTML.

Reply:
@louischan-oursky  The border width of the component should be adjustable, therefore there should be a component level css variable. I meant the value of component level css variable can be `1.5px`, i.e.
```css
:root {
  --secondary-button-ring-width: 1.5px;
}
```

So the base variables `--ring-width-x` can be removed.

Reply:
`--secondary-btn__ring-width`, to be exact.